src/Controllers/usersController.ts

```diff
@@ -0,0 +1,109 @@
+import { Request, Response } from 'express';
+import db from './../db';
+import { v4 as uuidv4 } from 'uuid';
+import { validationResult } from 'express-validator';
+
+export const createUser = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+  try {
+    const id = uuidv4();
+    console.log("id", id);
+    const login = req.body.login;
```

Comment:
Unfortunately, express doesn't provide you any architecture requirements.
I mean, you can write all your code inside index.ts file.
My suggestion is to create Users.model, that model does all operations witch related to actions with dateBase.
And your controller's work is the following, it checks request on validity, and if everything is good, invokes UsersModel.createUser().
In you model, you can make requests, and everything you want according to the database.